src/sentry/snuba/referrer.py

```diff
@@ -574,6 +572,50 @@ class ReferrerBase(Enum):
         "tasks.process_projects_with_sessions.session_count"
     )
     TRANSACTION_ANOMALY_DETECTION = "transaction-anomaly-detection"
+
+    TSDB_MODELID_0 = "tsdb-modelid:0"
```

Comment:
should we add a test to make sure that all of TSDBModel is included in here now?